web/src/components/ContactItem.tsx

```diff
@@ -39,10 +39,12 @@ const ContactItem: React.FC<ContactItemProps> = ({
   link,
   content
 }: ContactItemProps): ReactElement => (
-  <ContactContainer>
-    <Marker src={iconSrc} alt={iconAlt} />
-    <CleanLink to={link}>{content}</CleanLink>
-  </ContactContainer>
+  <CleanLink to={link}>
+    <ContactContainer>
+      <Marker src={iconSrc} alt={iconAlt} />
+      {content}
+    </ContactContainer>
+  </CleanLink>
```

Comment:
Good point. I moved the corresponding styles to the `StyledCleanLink` component, since it's not used anywhere else. However, the theme is not passed as a prop to `StyledCleanLink` in tests, for a reason unknown to me. (Result in the browser looks to be OK though!)

Would appreciate some feedback!

Reply:
Resolved, see below